README.md

```diff
@@ -57,7 +57,11 @@ Current Supported Tags:
     cf logs fluentbit-drain --recent
     ```
 
-7. If you are using an egress proxy, set the $HTTPS_PROXY variable. (TODO; current .profile assumes a $PROXYROUTE in the app's env)
+7. If you are using an egress proxy, set the PROXYROUTE variable and restage the app.
+    ```sh
+    cf set-env fluentbit-drain PROXYROUTE 'https://user:pass@myproxy.app.internal:61443'
+    cf restage fluentbit-drain
```

Comment:
A `restart` should be sufficient, since that variable is not used during staging at all.

Reply:
Done!